backend/utils/room_utils.py

```diff
@@ -157,5 +157,5 @@ async def remove_room_member(org_id: str, room_data: dict, member_id: str) -> di
 
     response = update_room["data"]
     response["member_id"] = member_id
-    response["room_id"] = room_data["_id"]
+    response["room_id"] = room_id
```

Comment:
We can also do something like this:

`response.update({"member_id": member_id, "room_id": room_id})`